tests/5.1/flush/test_flush_seq_cst.c

```diff
@@ -0,0 +1,45 @@
+//===---- test_flush_seq_cst.c ----------------------------------------------===//
+// 
+// OpenMP API Version 5.1 Nov 2020
+// 
+// This is a test of the flush directive with no memory-order-clause specified.
```

Comment:
Too verbose - don't need 5-8. If the test is based off another test in the V&V, you don't need to mention Examples doc.